Add docs for opensearch docs (#151)
perses/website: docs/blog/posts/v0.54.0.md

      {
        "kind": "Folder",
        "name": "infrastructure",
        "items": [
          { "kind": "Dashboard", "name": "nodeexporter" }
        ]
      }
    ]
  }
}
```

## New plugins

TODO

### OpenSearch

> This version of Perses now ships a bundled OpenSearch datasource along with a LogQuery plugin. With just a few clicks, OpenSearch users can query and view their logs directly in a Perses dashboard, right next to the metrics and traces they already have. Both plugins are part of the Perses distribution, so there is nothing extra to install, the current Perses release already has them built in.

Reviewer: ```suggestion
This version of Perses now ships a bundled OpenSearch datasource along with a LogQuery plugin. Users can now query and view their logs directly in a Perses dashboard, right next to the other panels with metrics and traces.
```

![OpenSearch datasource configuration](../../assets/images/blog/v054/opensearch-datasource.png)

Logs are queried with [PPL (Piped Processing Language)](https://opensearch.org/docs/latest/search-plugins/sql/ppl/index/). You add a **Logs Table** panel, pick the **OpenSearch Log Query** query type, point it at an index pattern, and write a PPL query — for example, showing the most recent `ERROR` lines from a service.

![OpenSearch Log Query panel editor](../../assets/images/blog/v054/recent-logs.png)

Results are rendered as log entries, with every other column in the row kept as a label so you can filter and display fields like `service.name`, `severityText`, or `traceId`. You can drop several log panels on a single dashboard. For instance, a broad "Recent Logs" view alongside a filtered "Error Logs" one.

![OpenSearch log panels on a dashboard](../../assets/images/blog/v054/log-dashboard.png)

> A couple of things happen automatically to make this feel native. Queries are bound to the dashboard's current time range. The plugin injects a timestamp filter for you, so the logs follow the time picker (you can turn this off per query). And because OpenSearch indices are schema-flexible, the plugin auto-detects the timestamp and message fields from common names, with per-query overrides when your index uses different ones. Since `traceId` is just another field, you can even sit an OpenSearch logs panel next to a Tempo or Jaeger trace panel on the same dashboard and pivot from a trace straight to the logs it produced.

Reviewer: ```suggestion
A couple of things happen automatically to make this feel native. Queries are bound to the dashboard's current time range. The plugin injects a timestamp filter for you, so the logs follow the time picker (you can turn this off per query). And because OpenSearch indices are schema-flexible, the plugin auto-detects the timestamp and message fields from common names, with per-query overrides when your index uses different ones. Since `traceId` is just another field, you can even setup an OpenSearch logs panel next to a Tempo or Jaeger trace panel on the same dashboard and pivot from a trace straight to the logs it produced.
```
